1-js/01-getting-started/2-manuals-specifications/article.md

```diff
@@ -1,42 +1,41 @@
 
-# Manuals and specifications
+# கையேடுகள் மற்றும் திட்ட விவரம்
 
-This book is a *tutorial*. It aims to help you gradually learn the language. But once you're familiar with the basics, you'll need other sources.
+இந்த புத்தகம் ஒரு *தனிமுறை கல்வி குறிப்பு* ஆகும். இது படிப்படியாக நிரலாக்க மொழியைக் (Programming Language) கற்றுக்கொள்ள உங்களுக்கு உதவுவதை நோக்கமாகக் கொண்டுள்ளது. நீங்கள் அடிப்படைகளை அறிந்தவுடன், உங்களுக்கு பிற மூல பாடங்கள் தேவைப்படும்.
 
-## Specification
+## திட்ட விவரம் (Specification)
 
-[The ECMA-262 specification](https://www.ecma-international.org/publications/standards/Ecma-262.htm) contains the most in-depth, detailed and formalized information about JavaScript. It defines the language.
+[ECMA-262 திட்ட விவரம்](https://www.ecma-international.org/publications/standards/Ecma-262.htm) ஜாவா ஸ்கிரிப்ட் (JavaScript - ஜாவா அச்சுருஎழுத்து) பற்றிய மிக ஆழமான, விரிவான மற்றும் முறைப்படுத்தப்பட்ட தகவல்களைக் கொண்டுள்ளது. இது நிரலாக்க மொழியை வரையறுக்கிறது.
 
-But being that formalized, it's difficult to understand at first. So if you need the most trustworthy source of information about the language details, the specification is the right place. But it's not for everyday use.
+அவ்வாறு முறைப்படுத்தப்பட்டிருந்தாலும், நேரடியாக புரிந்துகொள்ள கடினமாக இருக்கும். எனவே, நிரலாக்க மொழி விவரங்களைப் பற்றிய மிகவும் நம்பகமான தகவல்களை தெரிந்து கொள்ள திட்ட விவர குறிப்புகளே சரியான இடமாகும். ஆனால் இவற்றை அன்றாட பயன்பாட்டிற்கு பயன்படுத்த முடியாது.
 
-A new specification version is released every year. In-between these releases, the latest specification draft is at <https://tc39.es/ecma262/>.
+ஒவ்வொரு ஆண்டும் ஒரு புதிய திட்ட விவரப் பதிப்புரு (version) வெளியிடப்படுகிறது. இந்த வெளியீடுகளுக்கு இடையில், அண்மை கால திட்ட விவர வரைவை காண <https://tc39.es/ecma262/> இங்கே செல்லவும்.
 
-To read about new bleeding-edge features, including those that are "almost standard" (so-called "stage 3"), see proposals at <https://github.com/tc39/proposals>.
+கிட்டத்தட்ட தர அளவுப்படுத்தப்பட்ட ("நிலை 3" என்று அழைக்கப்படுகிறது) மற்றும் புதிய முன்னணி பண்புகளை (bleeding-edge features) பற்றி படிக்க, இங்கே உள்ள புத்தாய்வுத்திட்டங்களை <https://github.com/tc39/proposals> பார்க்கவும். 
 
-Also, if you're in developing for the browser, then there are other specs covered in the [second part](info:browser-environment) of the tutorial.
+மேலும், நீங்கள் உலாவி (browser) உருவாக்கும் பணியில் இருப்பவராக இருந்தால் , உங்களுக்காக பிற திட்ட விவரங்கள் இந்த தனிமுறை கல்வி குறிப்பின் (tutorial) [இரண்டாம் பக்கத்தில் (second part)](info:browser-environment)  கொடுக்கப்பட்டுள்ளன.
```

Comment:
I believe this translates to "if you're developing a browser" and not "if you're in developing for the browser"
 
Google translation looks cool:

```
மேலும், நீங்கள் உலாவிக்காக உருவாக்குகிறீர்கள் என்றால், டுடோரியலின் [இரண்டாம் பகுதி (second part)](info:browser-environment) இல் மற்ற விவரக்குறிப்புகள் உள்ளன.
```

Reply:
I agree

Reply:
Please

Reply:
Please send me a "suggested change". I will do a "commit suggestion".

Reply:
```suggestion
மேலும், நீங்கள் உலாவிக்காக உருவாக்குகிறீர்கள் என்றால், டுடோரியலின் [இரண்டாம் பகுதி (second part)](info:browser-environment) இல் மற்ற விவரக்குறிப்புகள் உள்ளன.
```